app/src/main/java/com/orange/ouds/app/ui/tokens/TokenCategoryDetailScreen.kt

```diff
@@ -115,12 +152,32 @@ fun TokenCategoryDetailScreen(tokenCategory: TokenCategory) {
     }
 }
 
+@Composable
+private fun TokenIllustration(tokenProperty: TokenProperty, token: Token<Any>) = when (tokenProperty) {
+    is TokenProperty.BorderWidth -> BorderIllustrationBox(width = token.value as Dp)
+    is TokenProperty.BorderRadius -> BorderIllustrationBox(shape = RoundedCornerShape(token.value as Dp))
+    is TokenProperty.BorderStyle -> BorderIllustrationBox(style = token.value as OudsBorderStyle)
+    is TokenProperty.Opacity -> tokenProperty.Illustration(opacity = token.value as Float)
+    is TokenProperty.Elevation -> tokenProperty.Illustration(elevation = token.value as Dp)
+    is TokenProperty.SizeIconDecorative -> tokenProperty.Illustration(size = token.value as Dp)
+    is TokenProperty.SizeIconWithLabel -> tokenProperty.Illustration(size = token.value as Dp, token.name)
+    is TokenProperty.SpaceColumnGap, TokenProperty.SpaceFixed, TokenProperty.SpaceScaled -> SpaceIllustrationBox(
```

Comment:
Maybe Figma changed meanwhile, but there are several differences with our current UI:

- There is a header for each token property.
- Scaled spaces are both horizontal and vertical.
- Tokens with icon and arrow variants are separated into three sections, maybe this could easily be fixed by splitting the enum as suggested in another comment.
- Padding stack with arrow is not displayed in Figma but appears on screen and the arrow missing.
- Icons and arrows are missing in column gap with icon, column gap with arrow and column gap with icon.

<img width="307" alt="Capture d’écran 2024-10-30 à 12 00 45" src="https://github.com/user-attachments/assets/9b9889c7-72b6-4ea7-acbc-f706dede2388">

Reply:
As discussed together, I created a new issue for the headers: https://github.com/Orange-OpenSource/ouds-android/issues/179

Reply:
`SpacePaddingInlineWithImageIllustrationRow` could also be renamed to `SpaceWithImageIllustrationRow` to match `SpaceWithIconIllustrationRow` and `SpaceWithArrowIllustrationRow`.